django_netjsongraph/base/link.py

```diff
@@ -89,6 +91,14 @@ def json(self, dict=False, **kwargs):
     def send_status_changed_signal(self):
         link_status_changed.send(sender=self.__class__, link=self)
 
+    def last_status_change_diff(self):
```

Comment:
I came to the conclusion that we can simplify the implementation and remove this addition because calculating this for each node in a large graph would slow down the generation of the NetworkGraph even further to little advantage. By returning the `status_changed` field in the API frontend javascript implementations can calculate this value on the fly - and having the raw value is good enough for now.